### PR TITLE
added solution code and example cases for the problem

### PR DESCRIPTION
**Pull request by isamad0201**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/Tenet-Coding/ArtofCoding/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added cpp solution to the problem "NextGenTower Problem"

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C++/dp/NextGenTowerProblem.cpp [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+{2, 3},
+{3, 4},
+{5, 6},
+{7, 8},
+{9, 10},
+{10, 11}}
+
+Output:
+4
+**/
```
Comment (soumik2012) [inline C++/dp/NextGenTowerProblem.cpp:133]: Add a new line at EOF.

**Review by soumik2012 — COMMENTED** [review]
Little Change

File: C++/dp/NextGenTowerProblem.cpp (hunk 1 above)
Comment (isamad0201) [reply C++/dp/NextGenTowerProblem.cpp]: I have added a new line at EOF i.e I have made code total of 134 lines (blank line is added at EOF)

**Review by soumik2012 — APPROVED** [review]
LGTM 💯  Thank you  @SanthoshS20  for contribute to @Tenet-Coding

**Pull request merged**